docs/src/less/ds6/index.less

```diff
@@ -2,6 +2,10 @@
     display: none;
 }
 
+body {
+    background-color: #f8f8f8;
```

Comment:
Should we use variables to reference this color?

Reply:
Yeah, I think we will point to the new background color variable when it's added. See #4